INSTALL.md

```diff
@@ -21,11 +21,9 @@
 * heartbeat-devel (Heartbeat)
 
 ## Optional Build Dependencies
-* ncurses-devel
-* openssl-devel
-* libselinux-devel
-* systemd-devel
-* dbus-devel
+* ncurses-devel (interactive crm_mon)
+* systemd-devel (systemd resource support)
```

Comment:
That's rather just for the special handling in spec file.
Currently it's also required to have `configure` enable systemd support, but that can already be worked around with specifying `systemdunitdir=/path ./configure`.